app/src/main/res/layout/activity_calculator.xml

```diff
@@ -58,8 +59,47 @@
         android:layout_marginEnd="173dp"
         android:layout_marginRight="173dp"
         android:text="TextView"
-        app:layout_constraintBottom_toTopOf="@+id/btnPerformOperation"
+        app:layout_constraintBottom_toTopOf="@+id/btnPerformOperationAddiction"
         app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintTop_toBottomOf="@+id/textInputLayout2" />
+
+    <Button
+        android:id="@+id/btnPerformOperationSubtraction"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="8dp"
+        android:layout_marginLeft="8dp"
+        android:layout_marginBottom="8dp"
+        android:text="-"
```

Comment:
Maybe doing some internationalization here can be good.

Reply:
would it be necessary to internationalize buttons with arithmetic operations?

Reply:
resolved